homeassistant/components/evohome/config_flow.py

```diff
@@ -0,0 +1,31 @@
+"""Config flow to configure Evohome integration."""
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_SCAN_INTERVAL
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RamsesConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Config flow for Evohome."""
+
+    VERSION = 1
+
+    async def async_step_import(self, import_data: dict[str, Any]) -> ConfigFlowResult:
+        """Handle a flow initiated by configuration file."""
+
+        options = {}
+
+        if CONF_SCAN_INTERVAL in import_data:
+            scan_interval: timedelta = import_data.pop(CONF_SCAN_INTERVAL)
+            options[CONF_SCAN_INTERVAL] = int(scan_interval.total_seconds())
```

Comment:
I would put everything in options. Scan interval is not going to be allowed to be changed anyway when you add the flow (should be deprecated in yaml  as a separate change).